File: sdkwork-app-sdk-go/types/voice_speaker_create_form.go
package types

// 发音人创建请求
type VoiceSpeakerCreateForm struct {
	Name string `json:"name"`
	LocalName string `json:"localName"`
	Code string `json:"code"`
	FaceImageUrl string `json:"faceImageUrl"`
	Channel string `json:"channel"`
	Product string `json:"product"`
	Gender string `json:"gender"`
	AgeType string `json:"ageType"`
	Type string `json:"type"`
	Models []string `json:"models"`
	ChannelSpeakerId string `json:"channelSpeakerId"`
	Version string `json:"version"`
	Description string `json:"description"`
	Tags []string `json:"tags"`
}

File: sdkwork-app-sdk-csharp/Models/VoiceSpeakerCreateForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class VoiceSpeakerCreateForm
    {
        public string? Name { get; set; }
        public string? LocalName { get; set; }
        public string? Code { get; set; }
        public string? FaceImageUrl { get; set; }
        public string? Channel { get; set; }
        public string? Product { get; set; }
        public string? Gender { get; set; }
        public string? AgeType { get; set; }
        public string? Type { get; set; }
        public List<string>? Models { get; set; }
        public string? ChannelSpeakerId { get; set; }
        public string? Version { get; set; }
        public string? Description { get; set; }
        public List<string>? Tags { get; set; }
    }
}
